src/main/java/org/skife/jdbi/v2/BasicHandle.java

```diff
@@ -487,4 +490,20 @@ public void registerContainerFactory(ContainerFactory<?> factory)
     {
         this.containerFactoryRegistry.register(factory);
     }
+
+    @Override
+    public void setSqlObjectContext(Class<?> sqlObjectType, Method sqlObjectMethod) {
```

Comment:
Yeah. The issue is I need a way to make it so any `Query`, `Update`, etc created in the method call would inherit the sql object type and method.

An alternate approach I considered was to create a decorator `Handle` class, which calls `setSqlObjectType` and `setSqlObjectMethod` on every `Query`, `Update`, etc created by the decorated handle. Then wrap every `Handler` in `SqlObject` in a decorator which wraps the handle in a decorate before calling down to the base handler. It seemed a little heavy handed, but it didn't result in any new API.